src/domain/validate.ts

```diff
@@ -1,17 +1,23 @@
 import { MAX_MONTH, MIN_MONTH } from './constant';
 
-export const validNumber = (value: string) => {
-  return /^(?:[1-9][0-9]{0,2}|)$/.test(value);
-};
+export const validNumber = (value: string) => /^(?:[1-9][0-9]{0,2}|)$/.test(value);
```

Comment:
이 함수도 boolean을 반환하므로 아래의 네이밍처럼 일관성을 만들어주면 어떨까요? 

```suggestion
export const isValidNumber = (value: string) => /^(?:[1-9][0-9]{0,2}|)$/.test(value);
```